tests/fv3_conf/ccpp_control_run.IN

```diff
@@ -35,24 +35,24 @@ else
   fi
 fi
 
-cp    @[RTPWD]/${inputdir}/INPUT/aerosol.dat .
-cp    @[RTPWD]/${inputdir}/INPUT/co2historicaldata_201*.txt .
-cp    @[RTPWD]/${inputdir}/INPUT/sfc_emissivity_idx.txt .
-cp    @[RTPWD]/${inputdir}/INPUT/solarconstant_noaa_an.txt .
+cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/aerosol.dat .
+cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/co2historicaldata_201*.txt .
+cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/sfc_emissivity_idx.txt .
+cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/solarconstant_noaa_an.txt .
 if [ $OZ_PHYS_NEW = .T. ]; then
-  cp  @[RTPWD]/${inputdir}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
+  cp  @[INPUTDATA_ROOT]/${inputdir}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
 elif [ $OZ_PHYS_OLD = .T. ]; then
-  cp  @[RTPWD]/${inputdir}/INPUT/global_o3prdlos.f77 .
+  cp  @[INPUTDATA_ROOT]/${inputdir}/INPUT/global_o3prdlos.f77 .
 fi
 if [ $H2O_PHYS = .T. ]; then
-  cp  @[RTPWD]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
+  cp  @[INPUTDATA_ROOT]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
 fi
-cp    @[RTPWD]/${inputdir}/*grb .
-cp    @[RTPWD]/${inputdir}/*_table .
-cp    @[RTPWD]/${inputdir}/*configure .
+cp    @[INPUTDATA_ROOT]/${inputdir}/*grb .
+cp    @[INPUTDATA_ROOT]/${inputdir}/*_table .
+cp    @[INPUTDATA_ROOT]/${inputdir}/*configure .
 
 if [ $CPLWAV = .T. ]; then
-cp    @[RTPWD]/WW3_input_data/mod_def.* .
-cp    @[RTPWD]/WW3_input_data/@[SYEAR]@[SMONTH]@[SDAY]/ww3_multi.inp .
-cp    @[RTPWD]/WW3_input_data/@[SYEAR]@[SMONTH]@[SDAY]/rmp_src* .
+cp    @[INPUTDATA_ROOT]/WW3_input_data/mod_def.* .
```

Comment:
mod_def files change more frequently than other input changes, I think it would be good to keep the mod_defs in the baseline directory while the other files live in the INPUTDATA_ROOT directory.  This could be done as a second commit for follow up if desired.

Reply:
May I ask what makes mod_def files changed so frequently when ww3 source code is updated? Is there a way to make ww3 grid def not tied up with source code updates?

Reply:
The mod_def files pre-process grid and physics information.  The fix file they process does not change and should remain in the INPUTDATA_ROOT directory.  However, if a new feature is added, this will change the binary processed file.  While not every update of WW3 updates the mod_defs, they will certainly be updated more than any other fixed input file, which is why I would recommend putting this in the baseline directory.    A major rewrite/restructuring of WW3 would be required for these files to not need to occasionally be updated including optimization of the pre-processing that occurs in the generation of these mod_def files.

Reply:
This PR separates all input files (now located in $INPUTDATA_ROOT) from all output (history) files which we save and call 'baselines'. When the rt.sh is executed with '-c' command line flag (create baseline mode) tests are executed and all output files that we need to save as 'a new baseline' are saved in $NEW_BASELINE which is then moved to develop-YYYYMMDD.

Moving purely input data ( mod_def.* ) from $INPUTDATA_ROOT to a baseline (output) directory will break this clear separation of input vs output, and so rt.sh will need to know that some input data are in $INPUTDATA_ROOT and some are in develop-YYYYMMDD ($RTPWD). It will also need some mechanism of syncing (some) input data from baseline directory over to the new baseline directory, which will be the special case just for these files.

Reply:
All mod_def.* files are binary but I see they all start with header like:

`^@^@^AÏWAVEWATCH III MODEL DEFINITION FILE2020-06-18`

What's the meaning of this date 2020-06-18? Is this some sort of version number of the file? If it is will the 'new' version of the mod_def files have different (version) date? If they will, maybe we can store multiple 'versions' of the mod_def files in same input directory and have rt.sh use the one that specific version of WW3 submoudule needs. 
Then when WW3 is updated and it needs new mod_def file we can just add them (as a new version) to the current $INPUTDATA_ROOT, without creating entire new input data directory. Adding new files should not require new input-data-YYYYMMDD.

Reply:
@DusanJovic-NOAA the 2020-06-18 is coming from: https://github.com/NOAA-EMC/WW3/blob/develop/model/ftn/w3iogrmd.ftn#L70 which is the "version" number of that file.  I like your idea of just having WW3 version numbers so that we can add more files without having more and more directories.  That is also clean in terms of keeping the input files separate and all together.  The only wrinkle with that, is then my ambition of some day adding an automatic generator of that input file when creating the baseline directory doesn't really work in the way I originally was thinking about, but I'm sure something could be worked out for that too.       

@junwang-noaa the mod_def files contain a lot of information and sometimes you are just storing that a new feature is "false" and you are not using it but because it's now an option this file will change.  Sometimes it changes every commit, many times it does not.   You can see a description of all of the inputs that go into this file here: https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_grid.inp the mod_def will also change if you use a different switch file but are still using the same ww3_grid.inp input file.

Reply:
@JessicaMeixner-NOAA @junwang-noaa @DusanJovic-NOAA, If I understand correctly, the INPUTDATA_ROOT dir and its content will also change (slowly, not as frequently as the baseline), right? For example, adding new input files for new regression test cases, replacing old/obsolete files, etc. If that's the case, the WW3 mod_def.* files (typically change less frequently) can still sit in the INPUTDATA_ROOT dir. Whenever, they need to be updated, they can be updated in the INPUTDATA_ROOT dir as well. I assume the INPUTDATA_ROOT dirs will also have different versions (saying INPUTDATA_yyyymmdd).  Just a thought. May or may not be useful.